script.js

```diff
@@ -215,14 +215,20 @@ function showMangaDetails(manga, categories, source) {
       switch (manga.status) {
           case 1: return 'Ongoing';
           case 2: return 'Completed';
-          case 3: return 'Licenced';
+          case 3: return 'Licensed';
           case 4: return 'Publishing Finished';
           case 5: return 'Cancelled';
           case 6: return 'On Hiatus';
           default: return 'Unknown';
       }
```

Comment:
I really just spelled that wrong. 🤣🤣🤣

Reply:
Bruhh... I trusted you on that. The branch I'm working on also has it as `Licenced`.

Reply:
Wasn't me, the two references are spelled with S :clueless:
https://github.com/Animeboynz/Mihon-Backup-Viewer/pull/21#issuecomment-2171810705